src/assimp/SceneConverter.cpp

```diff
@@ -533,7 +533,10 @@ void SceneConverter::convert(const aiMaterial* material, vsg::DescriptorConfigur
         defines.insert("VSG_TWO_SIDED_LIGHTING");
     }
 
-    if (getColor(material, AI_MATKEY_BASE_COLOR, pbr.baseColorFactor) || hasPbrSpecularGlossiness)
+    aiShadingMode shadingMode;
+    material->Get(AI_MATKEY_SHADING_MODEL, shadingMode);
+
+    if (!(shadingMode & aiShadingMode_Phong || shadingMode & aiShadingMode_Blinn) && (getColor(material, AI_MATKEY_BASE_COLOR, pbr.baseColorFactor) || hasPbrSpecularGlossiness))
```

Comment:
oo that's a bit painful to parse to figure out what is going on in the if.  May be worth breaking it down into separate lines to make it more redable/easier to understand the intent.